### ISSUE TITLE
[bug:1439706] Change default name in gerrit patch

### ISSUE THREAD

**Issue by gluster-ant** [gluster/build-jobs#1]
URL: https://bugzilla.redhat.com/1439706
Creator: sarumuga@redhat.com
Time: 20170406T12:16:11

Description of problem:

IMHO, We should remove this default name "Anonymous Coward".

There are people who wish to remain Anonymous and not wish to reveal their name during contribution.

First time contributor don't want to see their name like this :( just because they avoided github naming convention.

Suggestions is to use patch owner name directly or anything else.

**Comment by gluster-ant** [comment]
Creator: sarumuga@redhat.com
Time: 20170522T06:38:40
Text: (In reply to Nigel Babu from comment #1)
> This situation arises when we do not have a patch owner name. This happens
> to people who do not set their name in Gerrit. So, as much as I'd love to
> set patch owner name, I can't. If you have suggestion for a different
> default, please feel free to suggest.


"Anonymous" should do.

**Comment by mscherer** [comment]
Yeah, I think anonymous coward is a reference to slashdot, but not a great one. As we are moving away from gerrit, is there something we should do on github ?
